application/helpers/questionIndexHelper.php

```diff
@@ -66,7 +66,7 @@ private function __construct($iSurveyId)
         $oSurvey = \Survey::model()->findByPk($this->iSurveyId);
         if ($oSurvey) {
             $this->indexType = $oSurvey->questionindex;
```

Comment:
`$oSurvey->aOptions['questionindex'];` ?

Reply:
Updated

Reply:
Updated!

Reply:
PS : i didn't really check, just see it can be ;)